elastic/kibana
# Multi field terms

Implements multi field terms agg config. Other bucket is supported, missing bucket is not.
Refactors common parts between terms and multi_terms into a common place.

To test the expression level:
Start `node scripts/functional_tests_server.js --config test/interpreter_functional/config.ts`
Go to the "Run pipeline app"
Do this in the console (check results and queries on the wire):
```
await window.runPipeline(`kibana_context timeRange={timerange from='2015-09-21T00:00:00Z' to='now+2y'}
          | esaggs index={indexPatternLoad id='90943e30-9a47-11e8-b64d-95841ca0b247'}
          aggs={aggMultiTerms id="0" enabled=true schema="bucket" fields="extension.keyword" fields="geo.dest" size=20 orderAgg={aggCount id="order" enabled=true schema="metric"}}
          aggs={aggCount id="1" enabled=true schema="metric"}`, null, {})
```

See test cases in https://github.com/elastic/kibana/pull/116928/files#diff-3113b5ed44b84abf6e4829a3bac5cd7982220cab058ff938defa26bef0ccb03b
